docker-compose.yml

```diff
@@ -93,6 +93,20 @@ services:
         aliases:
           - devm
 
+  image-manager:
+    image: dojot/image-manager:latest
+    restart: always
+    depends_on:
+      - mongodb
+      - postgres
+      - orion
+      - kafka
+    networks:
```

Comment:
I think we might want to remove this (the aliasing of device-manager to devm is something that is not particularly necessary, and was kept only as some services would, eventually make calls to it using that old alias)